.github/config.yml

```diff
@@ -0,0 +1,30 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!🚀⚡
+  
+  
+  Thank you and congrats🎉 for opening your very first issue in this project.Facial Expression Recognition Classifier Model is a project to detect human emotions using Machine Learning.
```

Comment:
@muthuannamalai12 I think we don't require  `Facial Expression Recognition Classifier Model is a project to detect human emotions using Machine Learning` at this point, because they will open an Issue only after knowing about the project